app.json

```diff
@@ -71,7 +73,7 @@
     "heroku-redis:hobby-dev",
     "papertrail",
     "rollbar",
-    "scout",
```

Comment:
no longer using scout?

Reply:
Yeah, Librato has more metrics and Skylight handles the slow query analysis better.